spec.html

```diff
@@ -27800,16 +27802,18 @@ <h1>Date.prototype.toTimeString ( )</h1>
 
       <emu-clause id="sec-date.prototype.toutcstring">
         <h1>Date.prototype.toUTCString ( )</h1>
-        <p>The following steps are performed:</p>
+        <p>The `toUTCString` method returns a String value representing the instance in time corresponding to this time value. The format of the String is based upon "HTTP-date" from RFC 7231, generalized to support the full range of times supported by ECMAScript Date objects. It performs the following steps:</p>
```

Comment:
can we make this link to the section below that has a URL?

Reply:
I followed the existing pattern for IETF RFC references (cf. https://ecma-international.org/ecma-262/9.0/#sec-uri-syntax-and-semantics and https://ecma-international.org/ecma-262/9.0/#sec-escape-string ), so I think any change to that would also belong in a separate PR.